tests/Unit/NumericalAlgorithms/Spectral/Test_SwshCollocation.cpp

```diff
@@ -142,7 +142,7 @@ SPECTRE_TEST_CASE("Unit.NumericalAlgorithms.Spectral.SwshCollocation",
   const Collocation<ComplexRepresentation::RealsThenImags>&
       precomputed_collocation = precomputed_spherical_harmonic_collocation<
           ComplexRepresentation::RealsThenImags>(collocation_maximum_l_max + 1);
-  ERROR("Failed to trigger ASSERT in an assertion test");
+  ERROR("Failed to trigger ERROR in an error test");
```

Comment:
The usual thing we do is to not have an `ERROR()` and not mark the test `[[noreturn]]`, but doing it this way instead seems fine.